audbackend/core/api.py

```diff
@@ -92,16 +93,17 @@ def access(
 
 
 def available() -> typing.Dict[str, typing.List[Base]]:
-    r"""List available backend instances.
+    r"""List available repositories.
 
     Returns a dictionary with
-    registered backend aliases as keys
+    backend alias name as key
     (see :func:`audbackend.register`)
-    and a list with backend instances as values
+    and a list with repositories
+    on that backend as value
```

Comment:
Not exactly.
At the moment it returns a list of tuples containing backend class, host path, repo name:
```python
>>> import audbackend
>>> audbackend.create("file-system", ".", "repo")
>>> audbackend.available()["file-system"]
[('audbackend.core.backend.filesystem.FileSystem', '.', 'repo')]
```

A list with repositories would look like this to me:
```python
>>> audbackend.available()["file-system"]
[Repository('repo', '.', 'file-system')]
```

Reply:
For me it is still not completely clear what should be the purpose of `audbackend.available()`.
If it should list available repsoitories it shouldn't return a dictionary, but simply a list of repository objects as those state anyway which abckend they are.

I think we introduced its current behavior to not only show us existing repositories, but also which backend alias names are linked to which backend classes. Otherwise I see no use at all of the returned dict.

Reply:
Right, the keys list all registered names, even for backends where have no "active" repositories.